hack/check_go_errors.py

```
@@ -0,0 +1,113 @@
+        except Exception as e:
+            exit_code = 1
+            total_errors += 1
+            print(f"::error file={path}::Error processing file: {e}")
+
+    if total_errors > 0:
+        # When running in GitHub Actions, a job summary provides a high-level
+        # overview of the linting results on the workflow run's summary page.
+        summary_file_path = os.getenv("GITHUB_STEP_SUMMARY")
+        if summary_file_path:
+            with open(summary_file_path, "a") as f:
+                f.write(f"### Go Error Message Linting\n\n")
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)